ColorStripe/Views/Device/DeviceView.swift

```diff
@@ -55,9 +55,11 @@ struct DeviceView: View {
                         modeSelectionIsPresented.toggle()
                     }.foregroundColor(.accentColor)
                 }
-                HStack {
-                    Text("Speed")
-                    Slider(value: $viewModel.state.speed, in: 0...1)
+                if let _ = viewModel.state.mode {
```

Comment:
I prefer this syntax, because we don't use mode inside if statement
```suggestion
                if viewModel.state.mode != nil {
```

Reply:
Done :)